extensions/amp-fit-text/1.0/component.js

```diff
@@ -78,8 +78,16 @@ export function FitText({
       contentClassName={classes.fitTextContentWrapper}
       {...rest}
     >
-      <div ref={measurerRef} className={classes.fitTextContent}>
-        <div ref={heightRef} className={classes.minContentHeight}>
+      <div
+        ref={measurerRef}
+        className={classes.fitTextContent}
+        part="container"
+      >
+        <div
+          ref={heightRef}
+          className={classes.minContentHeight}
+          part="content"
+        >
```

Comment:
Should we use the part attribute here as a way of selecting these elements for testing?

I have only seen them being used for CSS purposes so far and wondering if we should do this from a code cleanliness pespective?

Reply:
Good point, I revised the test to look for the classes instead and removed these additions. Thanks for the recommendation!

Reply:
Cool, thanks for making the change!